index.css

```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
+
+
+
+body{
+
+}
+
+nav{
+    height: 120px;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 3;
+}
+
+nav img{
+    margin-left: 50px;
+}
+
+.navRight{
+    float: right;
+    margin-right: 200px;
+    padding-top: 30px;
+}
+
+
+.dropdown{
+    float: right;
+    margin-right: 200px;
+    margin-top: 70px;
+    display: none;
+    background-image: url("button.png");
+    background-size: 100%;
+    height: 5px;
+    width: 5px;
+    padding: 16px;
+    border: none;
+}
+
+nav li{
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.liText{
+    font-family: "Arial Rounded MT Bold";
+    font-weight: bold;
+    text-transform: uppercase;
+    margin-right: 50px;
+    color: #000000;
+    text-decoration: none;
+    border-bottom: 3px solid transparent;
+    padding: 10px;
+    font-size: 13px;
+}
+
+.liText2{
```

Comment:
needs a better name or try and do without using a specific class

Reply:
wait is this just a repeat of all the stuff in the previous class? if so then just use 1 class for both and then select 
.dropdown a {
display: none;
}